src/components/CorporateCampaign/CampaignStatus.vue

```diff
@@ -89,6 +98,19 @@ export default {
 			default: null
 		}
 	},
+	computed: {
+		inContextClasses() {
+			return this.inContext ? 'tw-relative tw-bg-gray-50 tw-rounded' : '';
+		}
+	},
+	methods: {
+		handlePromoLinkClick() {
+			// Do nothing if not within the In-Context scenario
+			if (!this.inContext) return false;
+			// Refresh the page we are in context
+			window.location = window.location.origin + window.location.pathname;
```

Comment:
I believe this can be simplified to `window.location.reload();`

Reply:
Will have a look, need to make sure we don't reload with prior query params, hence the deliberate composition.